app/serializers/admin/user_serializer.rb

```diff
@@ -12,10 +12,10 @@ class Admin::UserSerializer < ActiveModel::Serializer
   )
 
   def created_at
-    object.created_at.iso8601 rescue nil
+    object.created_at.iso8601
```

Comment:
I know that rubocop wants us to remove `rescue nil` but in this case I think we need to make sure this doesn't leave us with uncaught exceptions being thrown